test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -193,23 +262,163 @@ def __init__(self):
+                )
+
+        x = torch.randn(2, 3, 4)
+        y = torch.randn(7, 8, 9)
+        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
+            self,
+            test(),
+            (x,),
+            additional_test_inputs=[(y,)],
+        )
+
+    @pytorch_test_common.skip_dynamic_fx_test(
+        "_aten_convolution_onnx: _add_attribute_to_torchscript_node()"
+        " parameter value=[None, None] violates type hint"
+        "typing.Union[float, int, str, bytes, typing.Sequence[float],"
+        " typing.Sequence[int], torch.Tensor], as [None, None]:"
+    )
+    def test_transpose_infer_shape(self):
+        class TransposeModule(torch.nn.Module):
             def __init__(self):
                 super().__init__()
-                self.sigmoid = torch.nn.Sigmoid()
+                self.conv = torch.nn.Conv2d(3, 1, 3, stride=2)
 
             def forward(self, x):
-                x = torch.ops.aten.add(x, 1.0, alpha=2.0)
-                return self.sigmoid(x)
+                x = self.conv(x)
+                return x.transpose(0, 1)
 
-        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(SigmoidAddModel(), (x,))
+        x = torch.randn(32, 3, 64, 64)
+        y = torch.randn(16, 3, 8, 64)
+        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
+            self,
+            TransposeModule(),
+            (x,),
+            additional_test_inputs=[(y,)],
+        )
+
+    @pytorch_test_common.skip_dynamic_fx_test("torch._dynamo.exc.TorchRuntimeError")
+    def test_squeeze_runtime_dim(self):
+        class Squeeze(torch.nn.Module):
+            def forward(self, d1, d2):
+                t = torch.zeros(d1[0], d2[0])
+                return t.squeeze(0)
+
+        d1 = torch.tensor([1])
+        d3 = torch.tensor([3])
+        d4 = torch.tensor([4])
+        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
+            self, Squeeze(), (d1, d4), additional_test_inputs=[(d3, d4)]
+        )
+        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
+            self, Squeeze(), (d3, d4), additional_test_inputs=[(d1, d3)]
+        )
+
+    @pytorch_test_common.skip_dynamic_fx_test(
+        "AssertionError: The values for attribute 'shape' do not match:"
+        " torch.Size([5, 6, 2]) != torch.Size([4, 4, 2]). Even symbolic "
+        "fx.graph can't get dynamic arguments from this Module."
+    )
+    def test_slice(self):
+        class DynamicSliceExportMod(torch.nn.Module):
+            def forward(self, x):
+                results = []
+                for i in range(4):
+                    results.append(x[: x.size(0) - i, i : x.size(2), i:3])
+                return tuple(results)
+
+        x = torch.rand(5, 5, 5)
+        y = torch.randn(6, 7, 8)
+        _run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
+            self,
+            DynamicSliceExportMod(),
+            (x,),
+            additional_test_inputs=[(y,)],
+        )
 
     def test_mutation(self):
```

Comment:
Please do me a favor and skip this one for dynamic too #98622  🤣

Reply:
Sure!

Reply:
Hmmm The one reported is actually not dynamic. I will skip the whole test in this PR, and put skip_ORT_version on it in the next PR.